src/kaocha/config.clj

```diff
@@ -19,12 +19,15 @@
 
 (defn replace-by-default [config k]
   (if-let [v (get config k)]
-    (if (#{:prepend :append} (meta v))
+    (if (some #{:prepend :append} (keys (meta v)))
```

Comment:
:+1: had to stare at this for a few seconds before realizing the old version is borked.